_robots/2024Robot.md

```diff
@@ -0,0 +1,57 @@
+---
+title: 2024 Robot
+subtitle: Shockwave 
+description: Crescendo Robot 2024
+robot_code: 2024
+layout: robot
+imageicon: "../images/robots/2024robot.png"
+image: "../../images/robots/2024robot.png"
+CompetitionLogo : "../../images/robots/Competition Logos/2024_Crescendo.svg"
+features:
+    - label: Two-time Innovation in Control winners
+      icon: fa-certificate
+    - label: Shooter wrist 120-degree herringbone rack motion
+      icon: fa-robot
+    - label: Dual Photon vision and Limelight image processing
+      icon: fa-crosshairs
+awardsWon: 2
+hide_hero: true
+Specifications:
+- item: MK4i L3 Swerve Drive with Kraken Drive Motors enables 18.9ft/s speed.
+- item: Differential shooter speeds cause Note spin for improved shooting stability.
+- item: 120-degree wrist range with herringbone rack and pinion prevents backlash.
+- item: Two-stage belt-in-tube elevator with Kevlar-reinforced belt.
+- item: Elevator carriage reaches a max height in 0.5 seconds.
+- item: Elevator lifts robot at end-of-match in 0.87 seconds.
+- item: Intake uses independent Falcon 500 motors for belt-driven rollers.
+- item: Magnetic encoders and CANCoders measure the swerve module steering position.
+- item: Orange Pi 5+ processors for AprilTag and object detection in conjunction with two Limelight 3g’s.
+- item: Photon vision is used for faster and fluid Note detection, while the Limelight 3g’s are used for accurate Apriltag detection.
```

Comment:
Capitalize G in Limelight 3G's

Reply:
https://github.com/FRCTeam1987/teambroncobots.com/pull/66/commits/5bc76c6f668f5b8458d8a69859a379a385aaf1bc